### DISCUSSION TITLE
Some simple UI issues

### DISCUSSION THREAD

**Discussion by claudioduarte** [rails/rails#51415] [Docs]
Hello 👋🏻! First of all, congratulations to the team on this important milestone for RoR Guides! Props to everyone involved! 🎉 

Skimming through some sections, I noticed a few simple UI issues that I wanted to point out:

1. The table of contents overlaps the footer in the **Active Record Query Interface** page:

![ARQI](https://github.com/rails/rails/assets/4729521/0bacbdb7-7af3-49e9-845f-585d4d0cabbb)

2. The Rails logo **gets shrunk** on smaller screens

![rails-logo](https://github.com/rails/rails/assets/4729521/1efd1ec6-34b5-4285-a91b-18c3dd795fb2)

3. I would recommend adding some padding to the footer for mobile devices

![footer](https://github.com/rails/rails/assets/4729521/fd959227-d31e-4005-8b59-a465181ddf12)

Keep up the good work! 💪🏻

**Comment by AmandaPerino** [comment]
Hi @claudioduarte thanks for the feedback! Number 1 is a known issue that we are currently tackling now, but I will pass on points 2 and 3. Thanks for taking the time to reach out!

**Reply by claudioduarte** [reply]
Great! If I see anything else, I'll add it to this thread.

**Comment by AmandaPerino** [comment]
Hi @claudioduarte - going to close this comment out now that we have addressed all of these points in #51444. If you find more, please feel free to start a new discussion. Thanks again!

**Reply by claudioduarte** [reply]
Sure thing!
